Frontend/android-native/app/src/user/nammaYatri/AndroidManifest.xml

```diff
@@ -22,6 +22,7 @@
                 <data android:scheme="http"/>
                 <data android:host="nammayatri.in"/>
                 <data android:path="/u" />
+                <data android:path="/refer" />
```

Comment:
This can be added in main AndroidManifeast

Reply:
it can't be